website/pages/en/cookbook/subgraph-composition.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Subgraph Composition with Sushiswap v3 and Base
+---
+
+Use subgraph composition to create a base subgraph containing core data, facilitating the building and scaling of additional subgraphs on top.
+
+## Overview
+
+This guide uses the Sushiswap v3 subgraph on Base chain.
+
+### Benefits of Composition
+
+Subgraph composition is a powerful feature for scaling, enabling the reuse of existing data. It allows you to:
+
+- Speed up your subgraph's syncing speed
+- Handle errors and optimize the resync
+- Enhance your subgraph development to effortlessly scale
+
+> Note: You can use existing subgraphs, but they must be redeployed. This does not require you to write new code. However, you might want to separate your subgraph, which could require additional adjustments to development.
```

Comment:
Existing subgraphs must be redeployed with a newer spec version ( the exact version is still to be determined) we can add that in once thats finalized